CHANGELOG.md

```diff
@@ -99,7 +105,8 @@ which would improve the latency significantly.
 
 
 - Bumped `Wasmtime` version to `23.0.2`
- [#12011](https://github.com/Kong/kong/issues/12011)
+ [#12930](https://github.com/Kong/kong/issues/12930)
```

Comment:
I checked the original PR, but neither of them bumped the version to `23.0.2`.

Reply:
fixed